project/sites/pacni/themes/pacni_theme/src/scss/components.scss

```diff
@@ -7,6 +7,7 @@
 
 #header {
   background: resolve('header_bg.jpg') no-repeat top right;
+  margin: 5rem auto .1rem;
```

Comment:
I think this was supposed to be .5rem and not 5rem?

Reply:
Thanks this has been updated now